dev-python/lark-parser/lark-parser-0.7.8.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python module to propose a modern general-purpose parsing library for Python"
+HOMEPAGE="https://github.com/lark-parser/lark"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
Above notes have been mostly semantics but now this is real stuff: You can't keyword arm64 on a new package. It has to be tested and keyworded by a member of arm64 arch team.

Reply:
:+1: please drop arm64 for now. we need to file a bug with the arm64 arch team and ask them to keyword for ARM.